RecoTauTag/TauTagTools/test/training/skimming/buildBackgroundDataSet_cfg.py

```diff
@@ -309,7 +309,7 @@
 )
 
 # Build pizeros for our final jet collection
-import RecoTauTag.RecoTau.RecoTauPiZeroProducer_cfi as PiZeroProd
+import RecoTauTag.RecoTau.RecoTauPiZeroProducer_cff as PiZeroProd
 process.backgroundJetsRecoTauPiZeros = PiZeroProd.ak5PFJetsRecoTauPiZeros.clone(
```

Comment:
@swozniewski @mbluj (FYI): I have fixed the name of the imported config file, but this ak5PFJetsRecoTauPiZeros is nonetheless non existing any more in that file, since February 2014. I imagine that these tests are outdated: maybe at some point one could decide to also either update or remove them.